addons/l10n_ch/views/setup_wizard_views.xml

```diff
@@ -7,11 +7,8 @@
             <field name="inherit_id" ref="account.setup_bank_account_wizard"/>
             <field name="arch" type="xml">
                 <field name="bank_bic" position="after">
-                    <field name="l10n_ch_show_subscription" invisible="1"/>
-                    <field name="l10n_ch_isr_subscription_chf" attrs="{'invisible': [('l10n_ch_show_subscription', '=', False)]}"/>
-                    <label for="l10n_ch_postal" string="ISR Client Identification Number" attrs="{'invisible': [('l10n_ch_show_subscription', '=', False)]}"/>
-                    <field name="l10n_ch_postal" nolabel="1" attrs="{'invisible': [('l10n_ch_show_subscription', '=', False)]}"/>
-                    <field name="l10n_ch_qr_iban" attrs="{'invisible': [('l10n_ch_show_subscription', '=', False)]}"/>
+                    <field name="l10n_ch_bank_is_ch_or_li" invisible="1"/>
+                    <field name="l10n_ch_qr_iban" attrs="{'invisible': [('l10n_ch_bank_is_ch_or_li', '=', False)]}"/>
```

Comment:
Shouldn't we rather add a computed country_id field on res.partner.bank directly in account to replace this l10n_ch_bank_is_ch_or_li field, and just check the code of that country here ? To me, this could happen in other l10n, and having a single field for everyone would make sense.

Reply:
there is already a `country` `Many2one` in `res_bank`, which wouldn't IMO fit the specific checks we have in this field. 
I think having two fields talking about countries in `res_bank` would add confusion and cause human errors down the road... What do you think ?